scylla-cdc/src/stream_reader.rs

```diff
@@ -202,6 +164,55 @@ impl StreamReader {
 
         Ok(())
     }
+
+    /// Reads the CDC log using exponential backoff to handle timeouts.
+    async fn read_log(
```

Comment:
Nitpick: `read_log` is a very generic name, similar to `fetch_cdc`. Please give the function below a more descriptive name.